### PR TITLE
New apps: Acode

### PR DESCRIPTION
**Pull request by sitiom**
* ~Amaze File Manager~
* ~ONLYOFFICE Documents~
* Add `Text Editors` section
  - Acode
  - Move `Editor` to this section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -225,6 +226,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 
 ### • File Manager
 
+* [**Amaze File Manager**](https://github.com/TeamAmaze/AmazeFileManager) <sup>**[[F-Droid](https://f-droid.org/packages/com.amaze.filemanager/)]**</sup>
```
Comment (offa) [inline README.md:229]: Amaze is no longer FOSS unfortunately and was removed therefore (see #217).

File: README.md [hunk 2]
```diff
@@ -422,6 +423,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Office
 
 * [**Collabora Office**](https://www.collaboraoffice.com/solutions/collabora-office-android-ios/)
+* [**ONLYOFFICE Documents**](https://www.onlyoffice.com/office-for-android.aspx)
```
Comment (offa) [inline README.md:426]: The project contained proprietary and tracking components on my last check and at a glance it's still the case (see #130).

File: README.md [hunk 3]
```diff
@@ -506,6 +508,11 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 * [**Auditor**](https://github.com/GrapheneOS/Auditor)
 * [**Extirpater**](https://gitlab.com/divested-mobile/extirpater) <sup>**[[F-Droid](https://www.f-droid.org/app/us.spotco.extirpater)]**</sup>
 
+### • Text Editors
```
Comment (offa) [inline README.md:511]: Makes sense now, thanks! :+1:

**Review by offa — CHANGES_REQUESTED** [review]
Thanks for your contribution.

I have added some comments.

File: README.md (hunk 1 above)
Comment (sitiom) [reply README.md:229]: Oops, should've checked

File: README.md (hunk 2 above)
Comment (sitiom) [reply README.md:426]: Should've checked this too

File: README.md (hunk 1 above)
Comment (offa) [reply README.md]: No problem :-)

**Review by offa — APPROVED** [review]

**Comment by offa** [comment]
Thank you! :-)

**Pull request merged**